src/modal-manager/ModalManager.jsx

```diff
@@ -0,0 +1,145 @@
+
+import styles from './ModalManager.module.scss';
+
+const disclosureType = 'modal';
+export { disclosureType };
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * The components to be rendered in the body of the ModalManager. These components will receive the
+   * disclosure capabilities through the DisclosureManger's context API.
+   */
+  children: PropTypes.node,
+  /**
+   * The component to render within the Modal above the disclosed content.
+   */
+  disclosureAccessory: PropTypes.element,
+  /**
+   * The Object (or function that returns an Object) that specifies the messages
+   * used to prompt the user when disclosure dismissal occurs when pending state
+   * is present.
+   *
+   * This prop should be provided once upon the initial mount of the component. Any subsequent changes
+   * to this prop value may not be reflected for existing disclosed modals.
+   */
+  navigationPromptResolutionOptions: navigationPromptResolutionOptionsShape,
+};
+
+const heightFromSize = {
+  tiny: 240,
+  small: 420,
+  medium: 600,
+  large: 870,
+  huge: 960,
+};
+
+const widthFromSize = {
+  tiny: 320,
+  small: 640,
+  medium: 960,
+  large: 1280,
+  huge: 1600,
+};
+
+class ModalManager extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.renderModal = this.renderModal.bind(this);
+  }
+
+  renderModal(manager) {
+    const {
+      children, disclosureAccessory, navigationPromptResolutionOptions, ...customProps
+    } = this.props;
+
+    const containerClassNames = cx([
+      'container',
+      customProps.className,
+    ]);
+
+    const classArray = ['modal-manager'];
+    const isFullscreen = manager.disclosure.isMaximized || manager.disclosure.size === availableDisclosureSizes.FULLSCREEN;
+    if (!isFullscreen) {
+      if (manager.disclosure.dimensions) {
+        classArray.push(`height-${manager.disclosure.dimensions.height}`, `width-${manager.disclosure.dimensions.width}`);
+      } else if (manager.disclosure.size) {
+        classArray.push(`height-${heightFromSize[manager.disclosure.size]}`, `width-${widthFromSize[manager.disclosure.size]}`);
+      }
+    }
+
+    const presentedDisclosureComponentKey = manager.disclosureComponentKeys[manager.disclosureComponentKeys.length - 1];
+    const presentedDisclosureComponentData = manager.disclosureComponentData[presentedDisclosureComponentKey] || {};
+    const headerDataForPresentedComponent = presentedDisclosureComponentData.headerAdapterData;
+
+    return (
+      <div {...customProps} className={containerClassNames}>
+        {manager.children.components}
+        <AbstractModal
+          isOpen={manager.disclosure.isOpen}
+          isFullscreen={isFullscreen}
+          classNameModal={cx(classArray)}
+          onRequestClose={() => {
+            manager.closeDisclosure();
+          }}
+          closeOnEsc
+          closeOnOutsideClick={false}
+          ariaLabel=""
```

Comment:
Should we be maintaining the aria-label / does this affect screen readers? or was this removed due to lack of translations? https://github.com/cerner/terra-framework/blob/master/packages/terra-modal-manager/src/ModalManager.jsx#L85

Reply:
I know the previous provided value "Modal" wasn't sufficient, and I don't know if even a translated string for "Modal" would be sufficient. It seems like this should be content-specific, but we have no way of setting this value from the disclosure API.

I could put the existing value back to keep things exactly the same and sync up with accessibility on a separate issue. But I don't think we want this to hold up the PR.

Reply:
Since we're using the existing ModalManager package now, this was not changed.